chartfx-math/src/main/codegen/de/gsi/math/MathGen.java

```diff
@@ -202,7 +200,7 @@ public double KolmogorovTest(int na, double[] a, int nb, double[] b, String opti
      * @param work see above
      * @return the computed result
      */
-    double KOrdStat(int n, double[] a, int k, int[] work) {
+    double kOrdStat(int n, double[] a, int k, int[] work) {
```

Comment:
Codacy found an issue: [The method 'kOrdStat(int, double, int, int)' has an NPath complexity of 1770, current threshold is 200](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)